### ISSUE TITLE
setup images for test set to be served locally

### ISSUE THREAD

**Issue by mnaydan** [Princeton-CDH/ppa-nlp#73]
- [x] get mogrify script from Rebecca
- [x] update image path in nginx config

**Comment by rlskoeser** [comment]
Here's where I documented the syntax I used before for running mogrify: https://github.com/Princeton-CDH/ppa-nlp/issues/39#issuecomment-2204495465 :
```sh
 mogrify -format jpg -resize 500 */*.TIF
 ```

This was for Gale content, hence the `.TIF`.

Maybe copy the image directories we want from tigerdata to local storage then run mogrify to resize them in place, replacing the existing file. Relative path and filenames need to be the same because they are referenced in the dataset loaded into prodigy.  Mogrify isn't installed on cdh-prodigy1 yet, but let's add it.

On the cdh-ansible side of things:
- here's the variable we're using to configure the path for ppa images served out by nginx: https://github.com/Princeton-CDH/cdh-ansible/blob/31b01c46265b1bea89bf937fdd4ce53ec32cab34/inventory/group_vars/prodigy/vars.yml#L24-L25
- to install imagemagick (mogrify), we can define a list of app-specific ubuntu dependencies to our prodigy host vars and specify `imagemagick`; here's an example in another app: https://github.com/Princeton-CDH/cdh-ansible/blob/31b01c46265b1bea89bf937fdd4ce53ec32cab34/inventory/group_vars/prosody/vars.yml#L81-L83 
  - I'm pretty sure this is a setup task, so to get the package installed you'll need to run the playbook with setup tasks enabled
